frontend/src/app/main/ui/ds/tooltip/tooltip.scss

```diff
@@ -25,8 +21,8 @@ $arrow-size: 11px;
 .tooltip-arrow {
   background-color: var(--color-background-primary);
   border-radius: var(--sp-xs);
-  width: $arrow-size;
-  height: $arrow-size;
+  width: $arrow-side;
```

Comment:
Please use logical properties (I suggest using both inline and block-size, but do as you prefer)